daemon/api.go

```diff
@@ -66,6 +66,12 @@ var api = []*Command{
 	assertsFindManyCmd,
 }
 
+// PolicyKit action names
+const sideloadSnapAction = "com.ubuntu.snappy.sideload-snap"
```

Comment:
`sideload` these days means install from somewhere different than the store and provide all the required assertions. I think you are referring to `developer` snaps here, right?